images/varnish-drupal/drupal.vcl

```diff
@@ -1,6 +1,7 @@
 vcl 4.0;
 
 import std;
+import dynamic;
 
 # set backend default
 backend default {
```

Comment:
dynamic is using dns to look up the backend at runtime rather than vcl compile; it is the premiss of #682.

I've added back the other two parameters in ab6614c8057ab5aca63